### DISCUSSION TITLE
Checking height of body to allow for automatic sizing

### DISCUSSION THREAD

**Discussion by OskarZyg** [vgalin/html2image#41] [Ideas]
Could we not just check how tall the `body` tag and then use that as our height?

**Comment by dmanexe** [comment]
Well, that depends on what your <body> is supposed to do. 

I have found it better to set the body to a min-height of `100vh`, if I am generating square images with html2image.

This `vh` unit is 1% of total viewport height. You can read more about Relative length units here: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units
